src/util/dev/server.ts

```diff
@@ -511,7 +512,9 @@ export default class DevServer {
           `filtered out ${allFiles.length - files.length} files`
       );
 
-      const { builders, errors } = await detectBuilders(files, pkg);
+      const { builders, errors } = await detectBuilders(files, pkg, {
+        tag: cliVersion.includes('canary') ? 'canary' : 'latest'
```

Comment:
I'd prefer that you use the `getDistTag()` function from https://github.com/zeit/now-cli/blob/801a70621f6dc7289b7b9ff5828f797d41d8de1c/src/util/get-dist-tag.ts

Reply:
Added it 👍